src/App.tsx

```diff
@@ -1,25 +1,102 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/ToDo';
+
+function getUserById(userId:number) {
+  return usersFromServer.filter(user => user.id === userId);
+}
```

Comment:
Better use `find` here, because `find` stops once user is found, `filter` always goes till the end